serverless/template.yml

```diff
@@ -9,7 +11,7 @@ Metadata:
     LicenseUrl: "LICENSE"
     ReadmeUrl: "DEVELOPER_README.md"
     HomePageUrl: "https://aws.amazon.com/timestream/"
-    SemanticVersion: "1.2.0"
+    SemanticVersion: "1.3.0"
```

Comment:
Should probably update this when we do a new release instead.

Reply:
Removed.